sos/plugins/opendaylight.py

```diff
@@ -29,12 +29,12 @@ def setup(self):
         if self.get_option("all_logs"):
             self.add_copy_spec([
                 "/opt/opendaylight/data/log/",
-                "/var/log/containers/opendaylight/",
+                "/var/log/containers/opendaylight/karaf/logs/",
```

Comment:
This change means *restriction* in logs collected - is that intentional? Currently (when `--all-logs` option is used), whole directory `/var/log/containers/opendaylight/` is collected. You request collecting just subdirectory here? (could be reasonable, just want to be sure it is the intention)

Reply:
Yes, this is intentional. All ODL logs are now under /var/log/containers/opendaylight/karaf/logs. For example, we would have:

/var/log/containers/opendaylight/karaf/logs/karaf.log
/var/log/containers/opendaylight/karaf/logs/karaf.log.1

Whereas before, all ODL logs would be:

/var/log/containers/opendaylight/karaf.log
/var/log/containers/opendaylight/karaf.log.1